src/components/interpolatedColorLegend.ts

```diff
@@ -68,7 +68,7 @@ export module Components {
      */
     public formatter(formatter: Formatter): InterpolatedColorLegend;
     public formatter(formatter?: Formatter): any {
-      if (formatter === undefined) {
+      if (formatter === null) {
```

Comment:
If this is a bug fix for `InterpolatedColorLegend` that seems to be separate from this pull request, please move it out into a separate pull request.